tests/testdata/extension/helper/FreqParamRule.json

```diff
@@ -2,7 +2,7 @@
     {
```

Comment:
Maybe rename file as "HotspotRule.json" is better?

Reply:
All fixed. Please check.